main.go

```diff
@@ -14,7 +14,339 @@
+	"github.com/networkservicemesh/sdk/pkg/networkservice/common/authorize"
+	registryclient "github.com/networkservicemesh/sdk/pkg/registry/chains/client"
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/jaeger"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+	"github.com/networkservicemesh/sdk/pkg/tools/log/logruslogger"
+	"github.com/networkservicemesh/sdk/pkg/tools/opentracing"
+	"github.com/networkservicemesh/sdk/pkg/tools/resetting"
+	"github.com/networkservicemesh/sdk/pkg/tools/spiffejwt"
+	"github.com/networkservicemesh/sdk/pkg/tools/token"
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/go-spiffe/v2/spiffetls/tlsconfig"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// Config - configuration for cmd-forwarder-ovs
+type Config struct {
+	Name                string        `default:"forwarder" desc:"Name of Endpoint"`
+	NSName              string        `default:"ovsconnectns" desc:"Name of Network Service to Register with Registry"`
+	BridgeName          string        `default:"br-nsm" desc:"Name of the OvS bridge"`
+	TunnelIP            string        `desc:"IP or CIDR to use for tunnels" split_words:"true"`
+	ConnectTo           url.URL       `default:"unix:///connect.to.socket" desc:"url to connect to" split_words:"true"`
+	MaxTokenLifetime    time.Duration `default:"24h" desc:"maximum lifetime of tokens" split_words:"true"`
+	ResourcePollTimeout time.Duration `default:"30s" desc:"device plugin polling timeout" split_words:"true"`
+	DevicePluginPath    string        `default:"/var/lib/kubelet/device-plugins/" desc:"path to the device plugin directory" split_words:"true"`
+	PodResourcesPath    string        `default:"/var/lib/kubelet/pod-resources/" desc:"path to the pod resources directory" split_words:"true"`
+	SRIOVConfigFile     string        `default:"pci.config" desc:"PCI resources config path" split_words:"true"`
+	PCIDevicesPath      string        `default:"/sys/bus/pci/devices" desc:"path to the PCI devices directory" split_words:"true"`
+	PCIDriversPath      string        `default:"/sys/bus/pci/drivers" desc:"path to the PCI drivers directory" split_words:"true"`
+	CgroupPath          string        `default:"/host/sys/fs/cgroup/devices" desc:"path to the host cgroup directory" split_words:"true"`
+	VFIOPath            string        `default:"/host/dev/vfio" desc:"path to the host VFIO directory" split_words:"true"`
+}
+
 func main() {
+	// ********************************************************************************
+	// setup context to catch signals
+	// ********************************************************************************
+	ctx, cancel := getTermSigalContext()
+	defer cancel()
+
+	setupLogger(ctx)
+
+	starttime := time.Now()
+
+	log.FromContext(ctx).Infof("executing phase 1: get config from environment (time since start: %s)", time.Since(starttime))
```

Comment:
Once again, as a matter of personal taste, I find this style: https://github.com/networkservicemesh/cmd-forwarder-vpp/blob/e9103cf238a4b5c8449ef04170c4ccf78cbce523/main.go#L111-L113 easier to read... you are more than welcome to have a differing opinion :)

Reply:
sure, fixed it accordingly without making changes to linter settings. hope its ok now.